packages/installer/upgrade.sh

```diff
@@ -84,50 +56,55 @@ find_upgrade_channel() {
 }
 
 mount_image() {
+    STATEDIR=/run/initramfs/isoscan
```

Comment:
I'm not satisfied with this, but it gets automatically mounted there after boot. Should be ok for the time being

Reply:
The isoscan script is really simple, we should be capable to provide our own. We could even avoid the "scan" functionality which looks for the iso image at any available device, there is no need for that, we know the device that holds the image

Reply:
Shall we look over a replacement for isoscan? maybe we can do in a separate PR I guess the changes to the upgrade scripts would be really minimal

Reply:
Yes we can do it in a separate PR, no hurries for now.